test/tests/modules/io/videoCaptureTests.js

```diff
@@ -45,5 +45,24 @@ module.exports = () => {
         expect(cap.get(cv.CAP_PROP_FRAME_HEIGHT)).to.equal(360);
       });
     });
+
+    describe('set', () => {
+      it('should set properties', () => {
+        const cap = new cv.VideoCapture(getTestVideoPath());
+        const wasSet = cap.set(cv.CAP_PROP_POS_MSEC, 1000)
+        expect(cap.get(cv.CAP_PROP_POS_MSEC)|0).to.equal(1001);
+        expect(wasSet).to.equal(true);
+      });
+    });
+    describe('setAsync', () => {
+      it('should set properties', () => {
```

Comment:
Just a minor remark  :), in async tests need to invoke a done callback here, otherwise mocha will not evaluate your expects provided in the callback of setAsync:

`it('should set properties', (done) => {`